xwiki-platform-core/xwiki-platform-refactoring/xwiki-platform-refactoring-default/src/main/java/org/xwiki/refactoring/internal/DefaultModelBridge.java

```diff
@@ -179,20 +179,26 @@ public boolean copy(DocumentReference source, DocumentReference destination)
 
     @Override
     public boolean delete(DocumentReference reference)
+    {
+        return delete(reference, false);
+    }
+
+    @Override
+    public boolean delete(DocumentReference reference, boolean skipRecycleBin)
     {
         XWikiContext xcontext = this.xcontextProvider.get();
         try {
             XWikiDocument document = xcontext.getWiki().getDocument(reference, xcontext);
             if (document.getTranslation() == 1) {
-                xcontext.getWiki().deleteDocument(document, xcontext);
-                this.logger.info("Document [{}] has been deleted.", reference);
+                xcontext.getWiki().deleteDocument(document, !skipRecycleBin, xcontext);
+                this.logger.info("Document [{}] has been deleted (trashed: [{}]).", reference, !skipRecycleBin);
```

Comment:
Maybe use the "recycle bin" terminology for consistency.

Reply:
`trashed` replaced by `to the recycle bin`(see https://github.com/xwiki/xwiki-platform/pull/1401/files#diff-29fdf5e5e3805db76947ad7e90902ac7R194)